graphql-server/src/middlewares/session.ts

```diff
@@ -0,0 +1,65 @@
+import { Express } from "express";
+import { Strategy } from "openid-client";
+const passport = require("passport");
+const session = require("express-session");
+import { buildProps } from "../utils/buildProps";
+import { getKeycloakClient } from "../utils/session";
+
+const props = buildProps();
```

Comment:
usually best not to call a function in the module scope like this.  files should be inert.  nothing happens unless something is called